advisor/src/main/kotlin/Advisor.kt

```diff
@@ -65,21 +65,25 @@ class Advisor(
             return ortResult
         }
 
-        val providers = providerFactories.map { it.create(config) }
-
         val results = sortedMapOf<Identifier, List<AdvisorResult>>()
 
         val packages = ortResult.getPackages(skipExcluded).map { it.pkg }
+        if (packages.isEmpty()) {
+            log.info { "Skipping advisor run as there are no packages to give advice for." }
```

Comment:
Is "skipping" now still the right term here? Because an `AdvisorRun` still gets added to the result. Maybe just simply "There are no packages to give advice for."